server/requestorserver/server.go

```diff
@@ -159,11 +159,22 @@ var corsOptions = cors.Options{
 	AllowedMethods: []string{http.MethodGet, http.MethodPost, http.MethodDelete},
 }
 
+func (s *Server) createRouters() (rootRouter, apiRouter *chi.Mux) {
+	rootRouter = chi.NewRouter()
+	rootRouter.Route(s.conf.ApiPrefix, func(r chi.Router) {
```

Comment:
When `s.conf.ApiPrefix` does not start with `/` this panics; this is better checked during validation of the configuration.

Reply:
The panic chi causes is already pretty clear:
```
% irma server  -a someapiprefix                         
[...]
panic: chi: routing pattern must begin with '/' in 'someapiprefix/*'
```

Reply:
True. But:
* Some cleanup code (eg. close database connections) may not get called,
* it breaks a pattern: in lots of other cases of incorrect input the server also does not panic, but instead emits an error message (in `Configuration.initialize()` in `server/requestorserver/conf.go`),
* to me, "panic" makes this look much scarier than it is.

Reply:
Ah, right, you rightly prefer an error over a panic.  I've added the check to the configuration initialisation function.